parking_fsm:
  ros__parameters:

    # laser scan callback
    scale: 700  # scale of on LiDAR data
    hough_threshold: 60 # threshold of detecting line by Hough Transformation

    # velocity
    lin_vel: 0.2  
    ang_vel: 0.2

    # tolerances
    centerline_error_tolerance: 100  # Tolerance for centerline_error
    distance_tolerance: 0.3  # Distance tolerance
    closest_line_angle_tolerance: 0.02
    marker_timeout_tolerance: 1.0 # duration(seconds) marker has not been detected
    yaw_tolerance: 0.5